packages/react-conformance/src/defaultTests.tsx

```diff
@@ -223,23 +231,18 @@ export const defaultTests: TestObject = {
     });
 
     it(`preserves component's default classNames (component-preserves-default-classname)`, () => {
-      if (!handledClassName) {
-        return; // don't run this test if the main className test failed
+      if (!handledClassName || !defaultClassNames?.length) {
+        return; // don't run this test if the main className test failed or there are no defaults
       }
-      const defaultEl = customMount(<Component {...requiredProps} />);
-      const defaultComponent = getComponent(defaultEl, helperComponents, wrapperComponent);
-      const defaultClassNames = defaultComponent.getDOMNode().getAttribute('class')?.split(' ') || [];
 
-      const el = customMount(<Component {...mergedProps} />);
-      const component = getComponent(el, helperComponents, wrapperComponent);
-      const classNames = (component.getDOMNode().getAttribute('class') || '').split(' ');
+      const result = render(<Component {...mergedProps} />, renderOptions);
+      const el = getTargetElement(testInfo, result, 'className');
+      const classNames = (el.getAttribute('class') || '').split(' ');
```

Comment:
Unfortunately that doesn't work (at least without a cast) without adding `dom.iterable` everywhere. :( I tried adding those in react-conformance itself, but then other converged packages started failing on the `type-check` step because they're also checking react-conformance. I think this is due to path aliases?